docsrc/index.rst

```diff
@@ -87,6 +116,7 @@ Python API Documenation
 * :ref:`torch_tensorrt_py`
 * :ref:`torch_tensorrt_logging_py`
 * :ref:`torch_tensorrt_ptq_py`
+* :ref:`torch_tensorrt_fx_dynamo`
```

Comment:
When rendering, this link does not exist. Should it be just `torch_tensorrt_dynamo_py`?